src/addons/manager/addonmanager.cpp

```diff
@@ -170,7 +170,7 @@ bool AddonManager::loadManifest(const QString& manifestFileName) {
         if (i.value().m_addon->enabled()) ++pos;
         continue;
       }
-      if (!enabled) {
+      if (!enabled && pos < count()) {
```

Comment:
I suspect the bug is elsewhere. I don't see why this should happen. Are you able to reproduce it consistently?
`count` impl should be in sync with this for-stmt.

Reply:
Nope, it's totally intermitting. Yeah I agree it's probably somewhere else. I can try to dig a little bit and come back here.

Reply:
Eh... let's close it. If I stumble on it again I can file a proper ticket and investigate as necessary.